settings.py

```diff
@@ -28,6 +28,10 @@
 SITE_ID = 1
 USE_TZ = True
 
+from django.utils.crypto import get_random_string
+chars = 'abcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*(-_=+)'
+SECRET_KEY = get_random_string(50, chars)
```

Comment:
For edx-submissions, this settings file is used only in the test suite.  You're right that we'll need to add this to `*.auth.json` in edx-platform.